gameClient.js

```diff
@@ -100,7 +100,7 @@
             $(`.playerMat[user=${g_userID}] #floorLineTrashCan`)
               .attr('type', tileType)
               .toggleClass('option')
-              .show();
+              .removeClass('hidden');
```

Comment:
you want to avoid jquery's `show/hide`, generally it toggles between `display: block` and `display: none` but if you want something like `display: flex`, jquery won't know about it.

show/hide was designed waaaay back before flexbox was a thing.